FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -210,40 +210,99 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                                <bibl><ptr target="bm:Turaiev1905Eustathii"/><citedRange unit="page">134-167</citedRange></bibl> and in
-                                <bibl><ptr target="bm:Turaev1906Ewostatewos"/><citedRange unit="page">71-92</citedRange></bibl>. 
-                                No. 14 corresponds to No. 17 in <bibl><ptr target="bm:Turaiev1905Eustathii"/><citedRange unit="page">167-177</citedRange></bibl> and in
-                                <bibl><ptr target="bm:Turaev1906Ewostatewos"/><citedRange unit="page">92-97</citedRange></bibl>.
-                                <locus from="127r" to="127v">No. 15</locus> being a miracle of ʾEwosṭātewos and <persName ref="PRS10048Warasin"></persName>
-                                similar to <bibl><ptr target="bm:ContiRossini1912Studi"/><citedRange unit="page">633-634</citedRange></bibl>
-                                but longer than that with an additional note on an encounter with a hermit.</note>
+                            <note>15 stories, indicated with rubricated words <foreign xml:lang="gez">ተአምሪሁ፡ ለአቡነ፡ ኤዎስጣቴዎስ፡</foreign>.</note>
                             <colophon xml:id="coloph2"><locus from="122ra" to="122rb" xml:lang="gez"/>ተጽሕፈት፡ ዛቲ፡ መጽሐፍ፡ በዘመነ፡ ዮሐንስ፡ 
                                 ወንጌላዊ፡ በ፸፻ወ፪፻፡ ፳ወ፰፡ ዓመተ፡ ዓለም፡ ወሠርቀ፡ ሌሊት፡ ፬፡ ወሠርቀ፡ መዓልት፡ ፳ወ፮፡ በዘመነ፡ ንጉሥነ፡ <persName ref="PRS2377Bakaffa">በካፋ፡</persName> 
                                 ወጳጳስነ፡ <persName ref="PRS6072Krestodo">ክርስቶዶሉ፡</persName> ወመምሕርነሂ፡ <persName ref="PRS00000">መቃርዮስ፡</persName> 
                                 ወቂሰ፡ ገበዝሂ፡ <persName ref="PRS00000">በዕዳ፡ ማርያም፡</persName> ወጸሐፊሃኒ፡ <persName ref="PRS00000">አርሳንዮስ፡</persName> ዘተድባበ፡ በመንግሥተ፡ ሰማያት፡ አሜን። 
                                 ዛቲ፡ መጽሐፍ፡ ተወጽነት፡ አመ፡ ረቡዑ፡ ለሐምሌ፡ ወተፈጸመት፡ በካልአይ፡ አመት፡ አመ፡ ፳ሀ፡ ለሕዳር፡ በዘመነ፡ <persName ref="PRS00000">ማቴዎስ</persName>።</colophon>
                             </msItem>
+                            <msItem xml:id="p1_i2.1">
+                                <locus from="113vb" to="114rb"></locus>
+                                <title ref="LIT6350MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.2">
+                                <locus from="114rb" to="114vb"></locus>
+                                <title ref="LIT6351MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.3">
+                                <locus from="114vb" to="115ra"></locus>
+                                <title ref="LIT6352MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.4">
+                                <locus from="115ra" to="115va"></locus>
+                                <title ref="LIT6353MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.5.1">
```

Comment:
here something is wrong with either the numbering or the nesting
2.4 may not be followed by 2.5.1 at the same level
if 2.5.1 and 2.5.2 are two subchapters of 2.5. you need a 2.5 around them., or it must be 2.5., 2.6 etc.

like in a book
chapter 1.1
chapter 1.2
chapter 1.3
chapter 1.4
chapter 1.5
chapter 1.6
etc, all at the same level

or 
chapter 1.1
chapter 1.2
chapter 1.3
chapter 1.4
chapter 1.5
-subchapter 1.5.1
-subchapter 1.5.2
chapter 1.6
etc

three digits after two => subchapter => sublevel
I do not know which is the structure you wanted to have